src/Microsoft.AspNetCore.Mvc.Core/breakingchanges.netcore.json

```diff
@@ -28,5 +28,9 @@
     "TypeId": "public class Microsoft.AspNetCore.Mvc.ModelBinding.Binders.BodyModelBinderProvider : Microsoft.AspNetCore.Mvc.ModelBinding.IModelBinderProvider",
     "MemberId": "public .ctor(System.Collections.Generic.IList<Microsoft.AspNetCore.Mvc.Formatters.IInputFormatter> formatters, Microsoft.AspNetCore.Mvc.Internal.IHttpRequestStreamReaderFactory readerFactory, Microsoft.Extensions.Logging.ILoggerFactory loggerFactory, Microsoft.AspNetCore.Mvc.MvcOptions options)",
     "Kind": "Removal"
+  },
+  {
+    "TypeId": "public class Microsoft.AspNetCore.Mvc.ModelBinding.JQueryFormValueProvider : Microsoft.AspNetCore.Mvc.ModelBinding.BindingSourceValueProvider, Microsoft.AspNetCore.Mvc.ModelBinding.IEnumerableValueProvider",
+    "Kind": "Removal"
```

Comment:
Is this a bug in API Check?

Reply:
This is a design restriction inherent in the string comparisons done, API Check compares
``` c#
Microsoft.AspNetCore.Mvc.ModelBinding.BindingSourceValueProvider, Microsoft.AspNetCore.Mvc.ModelBinding.IEnumerableValueProvider
```
with
``` c#
Microsoft.AspNetCore.Mvc.ModelBinding.JQueryValueProvider
```
and, seeing they don't match, complains loudly. Don't have many options here because `BindingSourceValueProvider` is a base class, not an implemented interface i.e. can't work around issue by re-listing the implemented interface.